# Bioinformatics scientist/PhD candidate

## Summary
I study alternative splicing in the _C. elegans_ genome by writing algorithms in Python to parse through genomic data. Using a simple stochastic model of the noisy background of alternative splicing, the not-so-simple nature of alternative splice site selection is revealed. I am working on refining the process of describing these instances of biolgically regulated splicing against the noisy background. I also have extensisve experience as a teaching assistant, which has helped me develop strong communication and organizational skills. Currently, I am looking for internship opportunites that will allow me to expand my skillset into NGS analysis or other areas of bioinformatics.

## Education
+ University of California, Davis. In-progress PhD, Integrative Genetics and Genomics. September 2019-present
+ California State University, Long Beach. BSc, Microbiology. September 2014-May 2019

## Skills
* Proficiency in Python and Unix
* Basic undstanding of Snakemake, RNA-seq analysis, and other bioinformatic methods like BLAST
* Soft skills including
  
## Research experience
__University of California, Davis:__ September 2019-present<br>
__Advisor:__ Dr. Ian Korf<br>
__Thesis:__ Modeling the stochastic nature of alternative splicing<br>

> High-throughput RNA-seq data reveals the presence of millions of low-abundance, seemingly useless splic isoforms. We believe that RNA splicing is largely a stochastic process, with the 'correct' isoforms simply being more likely to occur. To test this hypothesis, I have generated all possible combinations (APC) of splice iformors from a given gene sequence. Isoforms are then scored using prbalistic models. A position weight matrix scores the donor and acceptor sites, and a Markov model and length model scores the exons and introns. We believe poor predictions made by the APC model indicate interference of the stochasitc process by > other biological processes. I am currently examing gff files generated by the APC algorithm.

Project repo: https://github.com/icacedo/splicing

__California State University, Long Beach:__ June 2017-May 2019<br>
__Advisor:__ Dr. Jesse Dillon<br>
__Project title:__ Total coliforms collected during wet vs. dry weather and from protected vs. non-protected beaches do not show differences in the incidence of antibiotic resistand coliforms<br>

> Fecal coliform levels increase in local beach water after heavy rainfall, increasing the risk of illness for beach goers. Additionally, environmental antibiotic resistance is also of concern in these coliforms. I tested the incidence of multiple antibioitic resitant coliforms from southern California beachesa by performing growth experiments on agar with antibiotic disks. Zones of inhibition were measured and used to determine the incidence of multiple antibioitc resistance, which was quite low and not significantly different depending on site. However, over 50% of isolates were shown to be amplicillin resistant. The project lead was Rebecca Hernandez, a Master's student.

Project paper:<br>
Hernandez, R., Acedo, I., Dillon, J. G. 2020. Impact of wave action and rainfall on incidence and antibiotic resistance of total coliforms in Southern California beaches. J. Water Health. 18(5) DOI:10.2166/wh.2020.100

## Teaching assistantship experience
* Introduction to biology, Fall 2023, Fall 2021
* Introduction to genomics and bioinformatics Spring 2022, Spring 2021
* Structure and function of macromolecules, Spring 2023
  
## Honors and awards:
__CSULB BUILD Research Scholar:__ June 2017-May 2019
> NIH funded research training program for upper division students seeking a PhD level education. Curriculum included how to present research professionally and conduct research ethically, as well as completing a research project.

__CSULB BUILD Research Associate:__ June 2016-May 2017
> NIH funded research program for lower division students exploring laboratory research.


