# Bioinformatics scientist/PhD candidate

## Summary
I study alternative splicing in the _C. elegans_ genome by writing algorithms in Python to parse through genomic data. My goal is to develop a model of the noisy background of alternative splicing. Currently, I am looking for internship opportunites that will allow me to expand my skillset into NGS analysis.

## Education
+ University of California, Davis. In-progress PhD, Integrative Genetics and Genomics. September 2019-present
+ California State University, Long Beach. BSc, Microbiology. September 2014-May 2019

## Research experience
__University of California, Davis:__ September 2019-present<br>
__Advisor:__ Dr. Ian Korf<br>
__Thesis:__ Modeling the stochastic nature of alternative splicing<br>

> High-throughput RNA-seq data reveals the presence of millions of low-abundance, seemingly useless splic isoforms. We believe that RNA splicing is largely a stochastic process, with the 'correct' isoforms simply being more likely to occur. To test this hypothesis, I have generated all possible combinations (APC) of splice iformors from a given gene sequence. Isoforms are then scored using prbalistic models. A position weight matrix scores the donor and acceptor sites, and a Markov model and length model scores the exons and introns. We believe poor predictions made by the APC model indicate interference of the stochasitc process by > other biological processes. I am currently examing gff files generated by the APC algorithm.
https://icacedo.github.io/
__California State University, Long Beach:__ June 2017-May 2019<br>
__Advisor:__ Dr. Jesse Dillon<br>
__Project title:__ Total coliforms collected during wet vs. dry weather and from protected vs. non-protected beaches do not show differences in the incidence of antibiotic resistand coliforms<br>





